src/bindings/PyDP/algorithms/util.cpp

```diff
@@ -14,4 +15,6 @@ void init_algorithms_util(py::module& m) {
   util.def("default_epsilon", &dp::DefaultEpsilon);
   util.def("get_next_power_of_two", &dp::GetNextPowerOfTwo);
   util.def("qnorm", &dp::Qnorm);
+  util.def("mean",&dp::Mean<double>);
+  // util.def("hello", &dp:: )
```

Comment:
I think a helper function is needed to covert the python list to a vector. Is this code tested using pytest?

Reply:
yes I have added a test, you can check that in this commit

Reply:
@chinmayshah99 should I proceed to change these to have a wrapper function which will allow us to specify the type ?

Reply:
Yeah, please go ahead.